pkg/cmd/grafana-cli/commands/list_upgradeable_command.go

```diff
@@ -0,0 +1,47 @@
+package commands
+
+import (
+	"errors"
+
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/services"
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/utils"
+	"github.com/hashicorp/go-version"
+)
+
+func (cmd Command) listUpgradeableCommand(c utils.CommandLine) error {
+	pluginsDir := c.PluginDirectory()
+
+	localPlugins := services.GetLocalPlugins(pluginsDir)
+
+	remotePlugins, err := cmd.Client.ListAllPlugins(c.String("repo"))
+	if err != nil {
+		return err
+	}
+
+	anythingToUpgrade := false
+
+	for _, localPlugin := range localPlugins {
+		for _, p := range remotePlugins.Plugins {
+			remotePlugin := p
```

Comment:
`p` should just be renamed to `remotePlugin`

Reply:
This code is just stolen^Wborrowed from [pkg/cmd/grafana-cli/commands/upgrade_all_command.go](https://github.com/grafana/grafana/blob/205c6724/pkg/cmd/grafana-cli/commands/upgrade_all_command.go#L37-L47) so I think there is value in keeping them the same?

Reply:
I'm aware - but while we're in here making changes, we can always try leave the campground cleaner ™️ 😄

Reply:
I like that attitude but then we'd also have to change the other place and that dirties down this PR = D

Could do another PR for that, sure.